cloudformation-templates/zs_cc_cf_template_gwlb.yaml

```diff
@@ -42,27 +58,100 @@ Parameters:
     Description: >-
       The Http Probe Healthcheck Port for ZS Cloud Connectors. Allowed values
       are [80, 1024-65535]
-  EnableCrossZoneLB:
+  GwlbCrossZoneLbEnabled:
     Type: String
+    Description: "Select \"true\" to enable cross-zone loadbalancing (default), else \"false\""
+    Default: true
     AllowedValues:
-      - false
       - true
-    Default: true
-    Description: Enable Cross Zone GWLB operation
+      - false
+  GwlbHealthCheckInterval:
+    Type: Number
+    MinValue: 5
+    MaxValue: 300
+    ConstraintDescription: >-
+      Health Check Interval: Minimum 5 and maximum 300 seconds
+    Description: "Interval for GWLB target group health check probing, in seconds, of Cloud Connector targets"
+    Default: 20
+  GwlbHealthyThreshold:
+    Type: Number
+    MinValue: 2
+    MaxValue: 10
+    ConstraintDescription: >-
+      Healthy Threshold: Minimum 2 and maximum 10
+    Description: "The number of successful health checks required before an unhealthy target becomes healthy"
+    Default: 2
+  GwlbUnhealthyThreshold:
+    Type: Number
+    MinValue: 2
+    MaxValue: 10
+    ConstraintDescription: >-
+      UnHealthy Threshold: Minimum 2 and maximum 10
+    Description: "The number of unsuccessful health checks required before a healthy target becomes unhealthy"
+    Default: 3
```

Comment:
i ended up just hiding these prompts in asg. i don't think we want customers being able to change these values. not easily at least.

CcGwlbTargetGroup:
    Type: 'AWS::ElasticLoadBalancingV2::TargetGroup'
    Properties:
      HealthCheckPath: /?cchealth
      HealthCheckPort: !Ref ZscalerHttpProbePort
      HealthCheckProtocol: HTTP
      HealthCheckIntervalSeconds: 10
      HealthyThresholdCount: 2
      UnhealthyThresholdCount: 3